project_1/part_1.py

```diff
@@ -85,16 +86,16 @@ def img_avg(a_img):
 
     print(all_img)
     #pdb.set_trace()
-    plt.plot(X, all_img)
-    plt.show()
+    #plt.plot(X, all_img)
+    #plt.show()
 
-    #cv2.namedWindow('image', cv2.WINDOW_NORMAL)
-    #cv2.namedWindow('image1', cv2.WINDOW_NORMAL)
-    #cv2.imshow('image1', img1)
-    #cv2.imshow('image', nimg1)
-    #cv2.waitKey(0)
-    #cv2.destroyAllWindows()
+    log_all_img = [log(aimg1,10),log(aimg2,10),log(aimg3,10),log(aimg4,10),log(aimg5,10), log(aimg6,10), log(aimg7,10), log(aimg8,10)]
```

Comment:
Convention is to have a space right after the comma like log(aimg1, 10). Can you put one space after every comma.